docs/notebooks/beta_regression.ipynb

```diff
@@ -0,0 +1,1557 @@
+{
```

Comment:
<p>This can be deleted now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/442/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>